entity/entity.proto

```diff
@@ -31,18 +31,16 @@ message Entity {
   repeated Contact contacts = 5;
 }
 
-message EntityId {
-  string entity_id = 1;
-  string entity_name = 2;
-}
-
 message Address {
   bool is_main = 1;
-  string label = 2;
-  string zip = 3;
-  string country = 4;
-  string address_line1 = 5;
-  string address_line2 = 6;
+  bool is_remit_to = 2;
+  bool is_pay_to = 3;
+  string label = 4;
+  string zip = 5;
+  string state = 6;
+  string country = 7;
+  string address_line1 = 8;
+  string address_line2 = 9;
```

Comment:
Per the doc, do we need contact person here? or can we use the list of contacts from the Entity itself?

Reply:
@pstehlik what do you think?

Reply:
this is the c/o field that addresses have. so not really a "contact" on the other sense.
e.g.
```
Centrifuge GmbH
Philip Stehlik  <---- this one. that's part of the actual address
Glogauer Str 6
2 HH, Links, 2 OG
12053 Berlin
Germany
```